api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleServiceTest.java

```diff
@@ -61,6 +63,26 @@ void shouldCreateNewPaymentCycleForEligibleClaim() {
         assertThat(result.getEligibilityStatus()).isEqualTo(EligibilityStatus.ELIGIBLE);
         assertThat(result.getPaymentCycleStatus()).isEqualTo(PaymentCycleStatus.NEW);
         assertThat(result.getChildrenDob()).isEqualTo(datesOfBirth);
+        assertThat(result.getExpectedDeliveryDate()).isEqualTo(today);
+        assertThat(result.getTotalEntitlementAmountInPence()).isEqualTo(entitlement.getTotalVoucherValueInPence());
+        assertThat(result.getTotalVouchers()).isEqualTo(entitlement.getTotalVoucherEntitlement());
+    }
+
+    @Test
+    void shouldCreateNewPaymentCycleWithoutExpectedDueDateForEligibleClaimWithoutPregnancyVouchers() {
+        LocalDate expectedDeliveryDate = LocalDate.now().minusMonths(6);
+        Claim claim = aClaimWithExpectedDeliveryDate(expectedDeliveryDate);
+        List<LocalDate> datesOfBirth = List.of(LocalDate.now(), LocalDate.now().minusDays(2));
```

Comment:
Same as above.

Reply:
I've changed the EED to nine months in the future.